pkg/controller/podautoscaler/replica_calculator_test.go

```diff
@@ -348,6 +348,7 @@ func (tc *replicaCalcTestCase) runTest(t *testing.T) {
 	stop := make(chan struct{})
 	defer close(stop)
 	informerFactory.Start(stop)
+	informerFactory.WaitForCacheSync(stop)
```

Comment:
According to @googs1025's reference examples these should be changed to a `ctx.Done()` or equivalent, and we don't need to be creating and passing in a channel.

Ref:

- https://github.com/kubernetes/kubernetes/pull/126228/files#diff-4caf67d968fc1971b19326934df3dd450b6ca42fbd82031bf343d6480d99e7c6L6449

Reply:
That makes sense to me. 👍

Reply:
No problem , I'll adjust

Reply:
you don't need another wait for cache, do you?

just below you have 
```
	if !cache.WaitForNamedCacheSync("HPA", stop, informer.Informer().HasSynced) {
		return
	}
```

for the context comment, the other example already had a context, this does not have it so the channel usage loooks corrcet, what we want is to not leack goroutines, and this serves that purpose

Reply:
I understand. So closing this PR :)